pkg/kubelet/dockershim/docker_image.go

```diff
@@ -98,10 +98,15 @@ func (ds *dockerService) RemoveImage(image *runtimeapi.ImageSpec) error {
 			}
 		}
 		return nil
+	} else if err != nil && dockertools.IsImageNotFoundError(err) {
+		return nil
 	}
 
 	_, err = ds.client.RemoveImage(image.Image, dockertypes.ImageRemoveOptions{PruneChildren: true})
-	return err
+	if err != nil && !isImageNotFoundError(err) {
```

Comment:
@yujuhong Thanks for the review. I tried with this at beginning, while as @feiskyer pointed out in #44381
> dockertools is about to remove in next release, so we don't need to change its codes.

So I moved them here.

> line 96 also calls RemoveImage. Shouldn't you also check the error message there?

Seems image existence have been guaranteed by upper inspect image in line 93, do we need to consider image been removed in that time slice?